prep-javascript-dom-events/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Events</title>
+</head>
+
+<body>
+  <button class="hello-world-btn">Hello World!</button>
+  <script>
+    // Your code here
+    addEventListener('DOMContentloaded', myFunction);
+    function myFunction() {
+      console.log('Content Loaded');
+    }
+
+    window.addEventListener('mousemove', myFunction2);
+    function myFunction2() {
+      console.log('Mouse is Moving');
+    }
+
+    var $helloWorld = document.querySelector('button.hello-world-btn');
+
+    addEventListener('click', myFunction3);
+    function myFunction3() {
+      'click';
+      console.log('Hello World');
+    }
```

Comment:
Hi Ben, for the `'DOMContentLoaded'` event, you want to add the listener to the document.

```js
    document.addEventListener('DOMContentloaded', myFunction);
    function myFunction() {
      console.log('Content Loaded');
    }
```

And for the `'click'` event you want to listen to the button.

```js
    $helloWorld.addEventListener('click', myFunction3);
    function myFunction3() {
      'click';
      console.log('Hello World');
    }
```